views/shared/_created_by.gohtml

```diff
@@ -1,5 +1 @@
-{{if .Creator}}
-Created {{timeElapsed .DateCreated "en"}} by {{.Creator.FullName}}.
-{{else}}
-Created {{timeElapsed .DateCreated "en"}}.
-{{end}}
+Created {{formatTime .DateCreated nil "02/01/2006 15:04"}}{{if .Creator}} by {{.Creator.FullName}}{{ end }}.
```

Comment:
@nics @netsensei @nicolasfranck Anyone knows how to get this working with the default timezone in devcontainers? Passing on `nil` for now as the `.TimeZone` in the dict coming from the `.Yield` method is returning `interface {}` (which is an invalid timezone).

Also tried setting `BIBLIO_BACKOFFICE_TIMEZONE="Europe/Brussels"` in ".devcontainer/.env" but that didn't seem to do anything (and shouldn't be necessary [if it's the default](https://github.com/ugent-library/biblio-backoffice?tab=readme-ov-file#configuration)).

Reply:
`TZ=Europe/Brussels` should normally do something, and that is an environment variable that has an effect in any shell (set `export TZ=Europe/Brussels` and then do `date` in your shell). But normally the time zone should be set correctly in the destined docker environment. Any code that uses the "default time zone" should pick that up.

See also my comment here: https://github.com/ugent-library/biblio-backoffice/issues/935#issuecomment-1422568412